src/platform/user/profile/vet360/tests/components/Vet360EditModalErrorMessage.unit.spec.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { shallow } from 'enzyme';
```

Comment:
Just curious if it was a conscious decision to use shallow mounting instead of RTL for this test. I'm all in on RTL now but this is actually one of the few times I think doing a shallow mount is totally valid; I think it's fair to test that this component simply makes and properly configures an `AlertBox` vs testing what it actually renders in the DOM.

Reply:
Yes, it was a conscious decision. Without getting into too much detail, I've spent my fair share trying to get the tests configured and to pass in both RTL and Cypress, but after running into issues with either implementation I decided that this approach would be just as reliable and sufficient.